phoebe/solverbackends/solverbackends.py

```diff
@@ -2009,13 +2009,6 @@ def mpi_failed_samples_callback(result):
 
 
 class _ScipyOptimizeBaseBackend(BaseSolverBackend):
-    """
-    See <phoebe.parameters.solver.optimizer.nelder_mead>.
-
-    The run method in this class will almost always be called through the bundle, using
-    * <phoebe.frontend.bundle.Bundle.add_solver>
-    * <phoebe.frontend.bundle.Bundle.run_solver>
-    """
```

Comment:
moved this into the individual subclasses instead of in this unexposed base class